ibis/pandas/execution/window.py

```diff
@@ -467,7 +467,19 @@ def execute_series_dense_rank(op, data, **kwargs):
     return data.rank(method='dense', ascending=True).astype('int64') - 1
 
 
-@execute_node.register(ops.PercentRank, (pd.Series, SeriesGroupBy))
+@execute_node.register(ops.PercentRank, pd.Series)
 def execute_series_percent_rank(op, data, **kwargs):
+    # TODO(phillipc): Handle ORDER BY
```

Comment:
I guess you just copied this comment from the other function. If this is a proper TODO, please reference the GitHub issue. Otherwise don't copy the comment.

Reply:
yep .. not sure about that .. maybe there is a reason for this comment ... but I am happy to just remove that. thanks